core/controllers/suggestion.py

```diff
@@ -148,6 +148,26 @@ def post(self):
 
         self.render_json(self.values)
 
+    # TODO(#11735): Currently, only handles the translation suggestion
+    # updates. This needs to update to generally handle the suggestions.
+    @acl_decorators.can_update_suggestions
+    def put(self, suggestion_id):
+        """Handles PUT requests.
+
+        Raises:
+            ValidationError. The suggestion is already handled.
+        """
+        suggestion = suggestion_services.get_suggestion_by_id(suggestion_id)
+        if suggestion.status == 'accepted':
```

Comment:
Are we going to allow editing rejected suggestions?

Reply:
As per the docstring this handler should not allow updating handled (accepted/rejected) suggestion.

Reply:
Rejected one are also filtered.